clients/web/src/init.js

```diff
@@ -130,6 +130,43 @@ _.extend(girder, {
             opts.headers['Girder-Token'] = token;
         }
         return Backbone.ajax(opts);
+    },
+
+    login: function (username, password) {
+        var auth = 'Basic ' + window.btoa(username + ':' + password);
+
+        return girder.restRequest({
+            method: 'GET',
+            path: '/user/authentication',
+            headers: {
+                Authorization: auth
+            }
+        }).then(function (response) {
+            girder.events.trigger('g:login.success', response.user);
```

Comment:
For  backward compatibility let's make sure this still triggers `g:login` on success.